pokemon-super/src/main/java/nl/novi/Main.java

```diff
@@ -0,0 +1,428 @@
+package nl.novi;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.Scanner;
+
+public class Main {
+  //  First i will one super class pokemon
+  //  I will make 4 sub classes ( this wil serve as fire, water, grass, and electric)
+  //  These will in turn also serve out as charmender, squirtle, bulbasaur and pikachu etc
+  //  Later i will make more subclasses so serve out more pokemon - This would become combinedClasses that use composition
+  //  I will reinact them to make a fight manually
+  //  Later I will create the logic that you really can play the game
+
+  public static void main(String[] args) {
+
+    // This pokemons need to be put into an array and there I will make a ramdom  object
+    ElectricPokemon pikachu = new ElectricPokemon("Pikachu", 50, 100, 0, 50, 50, 30); // your current pokemon.
+
+    // List of pokemon to fight with
+    List<Pokemon> pokemons = new ArrayList<Pokemon>();
+    pokemons.add(new FirePokemon("Charmender", 50, 100, 0, 50, 50, 20));
+    pokemons.add(new WaterPokemon("Squirtle", 50, 100, 0, 50, 50, 20));
+    pokemons.add(new GrassPokemon("Bulbasaur", 50, 100, 0, 50, 50, 30));
+    // Combined pokemons ( 2 combined combinations possible)
+    pokemons.add(new CombinedPokemon("Volcanion", 50, 100, 0, 50, 50, 30, new FirePokemon("volcanion"),
+      new WaterPokemon("volcanion")));
+    pokemons.add(new CombinedPokemon("Ludicolo", 50, 100, 0, 50, 50, 20, new WaterPokemon("ludicolo"), new GrassPokemon("ludicolo")));
+
+    // Initiate Scanner object
+    Scanner scanner = new Scanner(System.in);
+    String input;
+    String[] choises = {"1", "2", "3"}; // This will be used for serving the 3 choises available in the 'regular' pokemons
+    String[] choisesCombined = {"1", "2", "3", "4", "5", "6"}; // This will be served for the combined pokemons
+
+    // Randomize pokemon
+    Random random = new Random();
+    int randomIndex = random.nextInt(pokemons.size());
+    // The pokemon that will be fight with.
+    Pokemon randomPokemon = pokemons.get(randomIndex);
+
+    //  - this game will just give a couple of choises:
+    //  - start the game( this will start a tournament of 3 matches)
+    //  - choose attack( you are only to play with pikachu for now)
+    //  - defend move to add defence against challenger
+    //  - power up move will add extra damage to challenger
+    //  - with every draw there will be a report of the damage being done and the current hp
+    //  - when there is a winner this will be printed out
+    //  - eventually when you win you continue to the next pokemons
+    //  - if you win 4 times in a row you wil win the tournament
+    //  - you will always get a new challenger based how many are stored in the List of pokemons.
+
+    // Start of the game
+    System.out.println("Start the game, press y, to exit press any other key.");
+    input = scanner.nextLine();
+    if (input.equals("y")) {
```

Comment:
Je hebt nu de logica die checkt of je wilt starten, anders stopt ie. Dit gebeurt echter maar 1 keer. Het zou logischer zijn om te checken of je wilt stoppen, maar dat de gebruiker ook op elk moment kan stoppen. Daarvoor kun je wellicht de "input.equals" in een while loop zetten in plaats van de "while(true)" die er nu staat.